soc/arm/nordic_nrf/Kconfig.defconfig

```diff
@@ -43,7 +43,7 @@ config GPIO
 	depends on SPI
 
 choice CORTEX_M_DEBUG_NULL_POINTER_EXCEPTION_DETECTION
-	default CORTEX_M_DEBUG_NULL_POINTER_EXCEPTION_DETECTION_DWT if TEST_ARM_CORTEX_M && !ARM_NONSECURE_FIRMWARE && CPU_CORTEX_M_HAS_DWT
+	default CORTEX_M_DEBUG_NULL_POINTER_EXCEPTION_DETECTION_MPU if TEST_ARM_CORTEX_M && !ARM_NONSECURE_FIRMWARE && CPU_CORTEX_M_HAS_DWT
```

Comment:
I think a better commit message than:
>  [nrf temphack] soc: nrf: Switch null-pointer detection option to MPU
>
> Can't get nrf_cc3xx_platform to initialize with DWT mode

would be highly appreciated by the next person who'll do an upmerge.

If I have to do the next upmerge then I have no idea if this temphack could potentially be dropped or not.
Or what indications I should look for, in order to know if this could be an upstream issue that suddenly has been fixed.

Reply:
We have discussed this with @ioannisg and we are going to comment this differently than the statement here

Reply:
done